knimin/lib/format.py

```diff
@@ -28,6 +32,62 @@ def format_epmotion_file(volumes, destination):
     return "\n".join(contents)
 
 
+def format_index_echo_pick_list(idx_layout, volume):
+    """Formats the contents of an echo indexing pick list file
+
+    Parameters
+    ----------
+    idx_layout: 2d numpy array of ints
+        The per well index information
+    vol : float
+        The v
+
+    Returns
+    -------
+    str
+        The contents of the echo pick list file
+    """
+    # Add the headers
+    contents = ['Source Plate Name,Source Plate Type,Source Well,'
+                'Concentration,Transfer Volume,Destination Plate Name,'
+                'Destination Well']
+    idx_layout = np.asarray(idx_layout, dtype=np.int)
+    rows, cols = idx_layout.shape
+
+    i5contents = []
+    i7contents = []
+    vol = "%.3f" % volume
+
+    for i in range(rows):
+        for j in range(cols):
+            if idx_layout[i, j] != 0:
+                idx_info = db.get_shotgun_index_information(idx_layout[i, j])
+                dest = "%s%s" % (chr(ord('A') + i), j + 1)
```

Comment:
This is going to be prone to an easy error, and is repeated. Can this be decomposed to:

```python
def well(i, j):
    return (chr(ord('A') + i), j + 1)
```